lib/mayaUsd/fileio/writeJobContext.cpp

```diff
@@ -195,7 +195,8 @@ SdfPath UsdMayaWriteJobContext::ConvertDagToUsdPath(const MDagPath& dagPath) con
     // write to the parent (transform) path instead.
```

Comment:
As the comment stated, we should only write to parent if the current node is a shape node